<!DOCTYPE html>
<html>

  <head>


  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta name="description" content="">
  <meta name="author" content="Gary">
  <link rel="canonical" href="/indexZH.html">
  <link rel="icon" href="favicon.ico" type="image/x-icon">
  <title>Zhang Lab</title>

  <!-- Bootstrap core CSS -->
  <link href="assets/css/bootstrap.min.css" rel="stylesheet">

  <!-- Custom fonts for this template -->
  <link href="assets/css/all.min.css" rel="stylesheet" type="text/css">
  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
  <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
  <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
  <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>

  <!-- Custom styles for this theme -->
  <!--<link href="assets/css/agency.min.css" rel="stylesheet">-->
  <link href="assets/css/agency.css" rel="stylesheet">
  

  <!-- Page container change top padding when nav shrinks -->
  
  

</head>


  <body id="page-top">

        <!-- Navigation -->
  <nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav" > 
    <div class="container">
	  <a class="navbar-brand js-scroll-trigger text-uppercase" href="#page-top"><img height="60" src="assets\img\logo.png"/></a>
      <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
        菜单
        <i class="fas fa-bars"></i>
      </button>
      <div class="collapse navbar-collapse" id="navbarResponsive">
        <ul class="navbar-nav text-uppercase ml-auto"><li class="nav-item"><a class="nav-link js-scroll-trigger" href="indexZH">主页</a></li>
		  <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#biographyZH">个人简介</a></li>
		  <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#researchZH">研究方向</a></li>
		  <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#portfolio">研究成果</a></li>
		  <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#teamZH">团队成员</a></li>
		  <li class="nav-item"><a class="nav-link js-scroll-trigger" href="index">English</a></li>
		  
        </ul>
      </div>
    </div>
  </nav>
  <!-- End Navigation -->

  <!-- Header -->
  <header class="masthead">
    <div style='height:110px;background:#212529'>

    </div>
  </header>
  <!-- End Header -->


<!-- Team -->
<section class="bg-light page-section" id="biographyZH">
<div class="container" >
  <div class="row">
	<div class="col-lg-12 text-center">
	  <!-- <h2 class="section-heading text-uppercase">TEAM MEMBERS</h2> -->
	  <h3 class="section-subheading text-muted"></h3>
	</div>
  </div>
  <div class="row" >
  
  	<div class="col-sm-12">
  	  <div class="team-member">
    		<img  style='float:left;margin-right:1%;margin-top:4%' src="assets/img/team/1.jpg" alt="" >


      </div>
      <div class="" style='maigin-left:2%'>
        <h1 style='color:navy'>张 健 博士</h1>
        <h2 class="text-muted" style>副教授</h2>
        <h4>天津大学医学工程与转化医学研究院</h4>

          地址: &nbsp;天津市南开区卫津路92号 天津大学第10教学楼404室<br>
          邮箱:  &nbsp;jian_zhang@tju.edu.cn <br>
          官网:  &nbsp;<a href="http://amt.tju.edu.cn/portal/teachers/getperson/id/121/sId/3/bId/2.html" style='color:blue'>http://amt.tju.edu.cn/portal/teachers/getperson/id/121/sId/3/bId/2.html</a>
          <br>
          <br>
          <br>
          <p>
            <h1>个人简介</h1>
            <p class='text-muted' style="font-size:20px">张健，理学博士，硕士生导师，现任天津大学医学工程与转化医学研究院副教授，入选“北洋学者”英才计划。本科毕业于南开大学计算机科学与技术专业，后获得军事医学研究院生物信息学博士学位，期间赴美国哈佛大学生物统计与计算生物学系联合培养。重点面向生物安全、传染病防控和肿瘤精准医疗的实际需求开展研究工作，研究方向包括生物医学大数据（分子组学数据）挖掘与建模、计算免疫学技术与方法等。现主持国家自然科学基金、天津大学北洋学者研究计划2个项目，参与国家科技重大专项、国防科技创新特区项目等5项纵向项目；在包括Nature Genetics, Genome Medicine在内的权威国际期刊发表论文14篇。</p>


      </div>


  	</div>
  


  </div>
  <div class="row">
	<div class="col-lg-8 mx-auto text-center">
	  <div class="large text-muted">
</div>
	</div>
  </div>
</div>
</section>
<!-- End Team -->

<!-- research -->
  <section class="page-section" id="researchZH">
    <div class="container">

      <div class="row">
        <div class="col-lg-12 ">
          <h2 class="section-heading text-uppercase">研究方向</h2>
          
        </div>
      </div>

      <div class="row ">
	  
        <div class="col-md-12">
          <ul>
            <li><h4 class="service-heading"><p>计算免疫学</p>
</h4></li>
          </ul>

          <div class="text-muted" style="font-size:20px"><p>T细胞和B细胞是适应性免疫系统的两个核心，均在其表面表达抗原特异性受体，即T细胞受体（TCRs）和B细胞受体（BCRs）。 TCR 和 BCR 的广泛多样性确保适应性免疫系统能够识别大量抗原，包括病原体、自身抗原、过敏原和肿瘤新抗原。 因此，分析和表征这些抗原特异性受体已成为基础和临床免疫学研究的重要组成部分。 我们有兴趣开发计算技术来研究 TCR/BCR 序列与表位结合特异性之间的联系，并探索它们在疾病检测和监测治疗反应方面的潜力。</p>
</div>
        </div>
	  
        <div class="col-md-12">
          <ul>
            <li><h4 class="service-heading"><p>多组学数据挖掘</p>
</h4></li>
          </ul>

          <div class="text-muted" style="font-size:20px"><p>“组学”是指在测定中测量所有或几乎所有目标分子空间，因此它们提供了生物系统的整体视图。 组学数据的高效处理和分析，尤其是从这些数据中提取和发现模式，给生物信息学和计算生物学带来了许多挑战。 作为具有计算机科学背景的计算生物学家，我对开发和应用新颖的机器学习和数据挖掘方法来应对这些挑战特别感兴趣。</p>
</div>
        </div>
	  
      </div>

    </div>
  </section>
<!-- End research -->

<!-- Portfolio Grid -->
<section class="bg-light page-section" id="portfolio">
  <div class="container">
    <div class="row">
      <div class="col-lg-12 ">
        <h2 class="section-heading text-uppercase"><p>研究成果</p>
</h2>
        <h3 class="section-sheading">
         <!-- <ul>
          <li>
</li>
         </ul> -->
        </h3>
      </div>
    </div>
    <div class="row">
	
      <div class="col-md-4 col-sm-6 portfolio-item">
        <a class="portfolio-link" data-toggle="modal" href="#p1">
          <div class="portfolio-hover">
            <div class="portfolio-hover-content">
              <i class="fas fa-plus fa-3x"></i>
            </div>
          </div>
          <img class="img-fluid" src="assets/img/portfolio/3.png" alt="">
        </a>
        <div class="portfolio-caption">
          <p class='pub-title' style='font-weight: bold;'>Effect of neoadjuvant chemotherapy on the immune microenvironment in gastric cancer as determined by multiplex immunofluorescence and T cell receptor repertoire analysis</p>
          <p class="text-muted">Xing X#, Shi J#, Jia Y#, Dou Y#, Li Z#, Dong B, Guo T, Cheng X, Li X, Du H, Hu Y, Jia S*, Zhang J*, Li Z*, Ji J*</p>
        </div>
      </div>
	
      <div class="col-md-4 col-sm-6 portfolio-item">
        <a class="portfolio-link" data-toggle="modal" href="#p2">
          <div class="portfolio-hover">
            <div class="portfolio-hover-content">
              <i class="fas fa-plus fa-3x"></i>
            </div>
          </div>
          <img class="img-fluid" src="assets/img/portfolio/1.png" alt="">
        </a>
        <div class="portfolio-caption">
          <p class='pub-title' style='font-weight: bold;'>Immune receptor repertoires in pediatric and adult acute myeloid leukemia</p>
          <p class="text-muted">Zhang J, Hu X, Wang J, Sahu A D, Cohen D, Song L, Ouyang Z, Fan J, Wang B, Fu J, Gu S, Sade-Feldman M, Hacohen N, Li W, Ying X*, Li B*, Liu X S*</p>
        </div>
      </div>
	
      <div class="col-md-4 col-sm-6 portfolio-item">
        <a class="portfolio-link" data-toggle="modal" href="#p3">
          <div class="portfolio-hover">
            <div class="portfolio-hover-content">
              <i class="fas fa-plus fa-3x"></i>
            </div>
          </div>
          <img class="img-fluid" src="assets/img/portfolio/2.png" alt="">
        </a>
        <div class="portfolio-caption">
          <p class='pub-title' style='font-weight: bold;'>Landscape of B cell immunity and related immune evasion in human cancers</p>
          <p class="text-muted">Hu X#, Zhang J#, Wang J, Fu J, Li T, Zheng X, Wang B, Gu S, Jiang P, Fan J, Ying X, Zhang J, Carroll M C, Wucherpfennig K W, Hacohen N, Zhang F, Zhang P, Liu J S*, Li B*, Liu X S*</p>
        </div>
      </div>
	
    </div>
  </div>
    <div align='center'>
      <a class="btn btn-primary btn-xl text-uppercase js-scroll-trigger" href="publicationZH" align='right'>查看更多</a>
    </div>
</section>
<!-- Portfolio Modals -->

  <div class="portfolio-modal modal fade" id="p1" tabindex="-1" role="dialog" aria-hidden="true">
    <div class="modal-dialog">
      <div class="modal-content">
        <div class="close-modal" data-dismiss="modal">
          <div class="lr">
            <div class="rl"></div>
          </div>
        </div>
        <div class="container">
          <div class="row">
            <div class="col-lg-8 mx-auto">
              <div class="modal-body">
                <!-- Project Details Go Here -->
                <p class="pub-title" style='font-weight: bold;font-size:30px;margin-bottom: 1rem;'>Effect of neoadjuvant chemotherapy on the immune microenvironment in gastric cancer as determined by multiplex immunofluorescence and T cell receptor repertoire analysis</p>
                <div class="item-intro text-muted"></div>
				<p><p style="margin-bottom: 0px;">Xing X#, Shi J#, Jia Y#, Dou Y#, Li Z#, Dong B, Guo T, Cheng X, Li X, Du H, Hu Y, Jia S*, <b>Zhang J*</b>, Li Z*, Ji J*</p>
<p>J Immunother Cancer. 2022 Mar;10(3):e003984.</p>

<p><b>Abstract</b><br />
<b>Background:</b> The combination of immune checkpoint blockade and chemotherapy has revolutionized the treatment of advanced gastric cancer (GC). It is crucial to unravel chemotherapy-induced tumor microenvironment (TME) modulation and identify which immunotherapy would improve antitumor effect.<br />

<b>Methods:</b> In this study, tumor-associated immune cells (TAICs) infiltration in residual tumor after neoadjuvant chemotherapy (NAC) together with 1075 cases of treatment-naïve GC patients was analyzed first. Then we performed multiplex fluorescence staining of a panel of immune markers (CD3, CD4, CD8, FOXP3 and PDL1) and T cell receptor β-chain sequencing to phenotype and enumerate T cell subpopulations and clonal expansion in paired GC samples (prechemotherapy and postchemotherapy) from another cohort of 30 cases of stage II/III GC patients.<br />

<b>Results:</b> Infiltration of CD68+ macrophages in residual tumors after NAC was significantly decreased compared with treatment-naïve GC patients, while no significant difference observed with respect to other immune markers. In residual tumors, post-NAC CD8 +T cells and CD68+ macrophages levels were significantly associated with chemotherapy response. Post-NAC CD8+ T cell levels remained as an independent predictor for favorable prognosis. Furthermore, when comparing the paired samples before and after NAC from 30 cases of stage II/III GC patients, we found FOXP3+ regulatory T cells proportion significantly decreased after chemotherapy. Pre-NAC FOXP3+ T reg cells level was much richer in the response group and decreased more significantly in the stromal compartment. CD8+ cytotoxic T lymphocytes levels were elevated after chemotherapy, which was more significant in the group treated with XELOX regimen and in patients with better response, consistent with the TCR diversity elevation.<br />

<b>Conclusions:</b> These findings have deepened our understanding of the immune modulating effect of chemotherapy and suggest that the immune profile of specimens after standard chemotherapy should be considered for the personalized immunotherapy to ultimately improve clinical outcome in patients with GC.<br />

</p>
</p>
                <button class="btn btn-primary" data-dismiss="modal" type="button">
                  <i class="fas fa-times"></i>
                  Close Paper</button>
                  <button class="btn btn-primary" type="button">
                      <a href="https://doi.org/10.1136/jitc-2021-003984" style='color:#FFF;text-decoration:none;' target='_blank' align='right'>Journal Page</a>
                  </button>

              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>

  <div class="portfolio-modal modal fade" id="p2" tabindex="-1" role="dialog" aria-hidden="true">
    <div class="modal-dialog">
      <div class="modal-content">
        <div class="close-modal" data-dismiss="modal">
          <div class="lr">
            <div class="rl"></div>
          </div>
        </div>
        <div class="container">
          <div class="row">
            <div class="col-lg-8 mx-auto">
              <div class="modal-body">
                <!-- Project Details Go Here -->
                <p class="pub-title" style='font-weight: bold;font-size:30px;margin-bottom: 1rem;'>Immune receptor repertoires in pediatric and adult acute myeloid leukemia</p>
                <div class="item-intro text-muted"></div>
				<p><p style="margin-bottom: 0px;"><b>J. Zhang</b>, X. Hu, J. Wang, A. Das Sahu, D. Cohen, J. Fan, B. Wang, J. Fu, S. Gu, M. Sade-Feldman, N. Hacohen, W. Li, X. Ying*, B. Li*, X.S. Liu*</p>
<p>Genome Med. 2019 Nov 26;11(1)73.</p>

<p><b>Abstract</b><br />
<b>Background:</b> Acute myeloid leukemia (AML), caused by the abnormal proliferation of immature myeloid cells inthe blood or bone marrow, is one of the most common hematologic malignancies. Currently, the interactionsbetween malignant myeloid cells and the immune microenvironment, especially T cells and B cells, remain poorlycharacterized.<br />
<b>Methods:</b> In this study, we systematically analyzed the T cell receptor and B cell receptor (TCR and BCR) repertoiresfrom the RNA-seq data of 145 pediatric and 151 adult AML samples as well as 73 non-tumor peripheral blood samples.<br />
<b>Results:</b> We inferred over 225,000 complementarity-determining region 3 (CDR3) sequences in TCRα,β,γ,andδchains and 1,210,000 CDR3 sequences in B cell immunoglobulin (Ig) heavy and light chains. We found higher clonalexpansion of both T cells and B cells in the AML microenvironment and observed many differences between pediatricand adult AML. Most notably, adult AML samples have significantly higher level of B cell activation and more secondaryIg class switch events than pediatric AML or non-tumor samples. Furthermore, adult AML with highly expanded IgA2 Bcells, which might represent an immunosuppressive microenvironment, are associated with regulatory T cells andworse overall survival.<br />
<b>Conclusions:</b> Our comprehensive characterization of the AML immunereceptor repertoires improved our understandingof T cell and B cell immunity in AML, which may provide insights into immunotherapies in hematological malignancies.</p>
</p>
                <button class="btn btn-primary" data-dismiss="modal" type="button">
                  <i class="fas fa-times"></i>
                  Close Paper</button>
                  <button class="btn btn-primary" type="button">
                      <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-019-0681-3" style='color:#FFF;text-decoration:none;' target='_blank' align='right'>Journal Page</a>
                  </button>

              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>

  <div class="portfolio-modal modal fade" id="p3" tabindex="-1" role="dialog" aria-hidden="true">
    <div class="modal-dialog">
      <div class="modal-content">
        <div class="close-modal" data-dismiss="modal">
          <div class="lr">
            <div class="rl"></div>
          </div>
        </div>
        <div class="container">
          <div class="row">
            <div class="col-lg-8 mx-auto">
              <div class="modal-body">
                <!-- Project Details Go Here -->
                <p class="pub-title" style='font-weight: bold;font-size:30px;margin-bottom: 1rem;'>Landscape of B cell immunity and related immune evasion in human cancers</p>
                <div class="item-intro text-muted"></div>
				<p><p style="margin-bottom: 0px;">Hu X#, <b>Zhang J#</b>, Wang J, Fu J, Li T, Zheng X, Wang B, Gu S, Jiang P, Fan J, Ying X, Zhang J, Carroll M C, Wucherpfennig K W, Hacohen N, Zhang F, Zhang P, Liu J S*, Li B*, Liu X S*</p>
<p>Nat Genet 51, 560–567 (2019).</p>

<p><b>Abstract</b><br />
Tumor-infiltrating B cells are an important component in the microenvironment with unclear anti-tumor impacts. We enhanced our previous computational algorithm TRUST to extract the B cell immunoglobulin (Ig) hypervariable regions from bulk tumor RNA-seq data. TRUST assembled over 30 million complementarity-determining region 3(CDR3s) of the B cell heavy chain (IgH)from The Cancer Genome Atlas (TCGA). Widespread B cell clonal expansions and Ig subclass switch events were observed in diverse human cancers. Prevalent somatic copy number alterationsin MICA and MICB genes related to antibody-dependent cell mediated cytotoxicity (ADCC) were identified in tumors with elevated B cell activity. IgG3-1 subclass switch interacts with the B cell receptor affinity maturation and defects in the ADCC pathway. Comprehensive pan-cancer analyses of tumor-infiltrating B cell receptor repertoires identified novel tumor immune evasion mechanisms through genetic alterations. The IgH sequences identified here are potentially useful resources for future development of immunotherapies.</p>
</p>
                <button class="btn btn-primary" data-dismiss="modal" type="button">
                  <i class="fas fa-times"></i>
                  Close Paper</button>
                  <button class="btn btn-primary" type="button">
                      <a href="https://www.nature.com/articles/s41588-018-0339-x" style='color:#FFF;text-decoration:none;' target='_blank' align='right'>Journal Page</a>
                  </button>

              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>

<!-- End Portfolio Modals -->


<!-- resource -->

<!-- End resource -->

<!-- Team -->
<section class="page-section" id="teamZH">
<div class="container">
  <div class="row">
	<div class="col-lg-12 text-center">
	  <h2 class="section-heading text-uppercase">团队成员</h2>
	  <h3 class="section-subheading text-muted"></h3>
	</div>
  </div>
  <div class="row">
  
	<!-- <div class="col-sm-4">
	  <div class="team-member">
		<img class="mx-auto rounded-circle" src="assets/img/team/2.jpg" alt="">
		<h4>豆允生</h4>
		<p class="text-muted">硕士生</p>
		<p>豆允生是天津大学2020级智能医学工程专业的研究生。他目前对开发计算方法来分析大规模TCR-seq数据感兴趣。本科就读与河海大学通信工程专业。 </p>
		<ul class="list-inline social-buttons">
		
		</ul>
	  </div>
	</div> -->
  <div class="col-sm-12">
    <div class="team-member" style='float:left;margin-right:10%'>
      <img class="mx-auto rounded-circle" src="assets/img/team/2.jpg" alt="" >
      <h4>豆允生</h4>
      <p class="text-muted">硕士生</p>
    </div>
      <p style='margin-top:5%;font-size:20px' >豆允生是天津大学2020级智能医学工程专业的研究生。他目前对开发计算方法来分析大规模TCR-seq数据感兴趣。本科就读与河海大学通信工程专业。 </p>
        <p align='left'><i class='fa fa-envelope' ></i> douys0130@tju.edu.cn</p>
  </div>
  
	<!-- <div class="col-sm-4">
	  <div class="team-member">
		<img class="mx-auto rounded-circle" src="assets/img/team/3.jpg" alt="">
		<h4>单世雯</h4>
		<p class="text-muted">硕士生</p>
		<p>单世雯是天津大学2021级智能医学工程专业的研究生。她在中国石油大学（北京）克拉玛依校区获得了软件工程学士学位。</p>
		<ul class="list-inline social-buttons">
		
		</ul>
	  </div>
	</div> -->
  <div class="col-sm-12">
    <div class="team-member" style='float:left;margin-right:10%'>
      <img class="mx-auto rounded-circle" src="assets/img/team/3.jpg" alt="" >
      <h4>单世雯</h4>
      <p class="text-muted">硕士生</p>
    </div>
      <p style='margin-top:5%;font-size:20px' >单世雯是天津大学2021级智能医学工程专业的研究生。她在中国石油大学（北京）克拉玛依校区获得了软件工程学士学位。</p>
        <p align='left'><i class='fa fa-envelope' ></i> shan_shiwen@163.com</p>
  </div>
  
  </div>
  <div class="row">
	<div class="col-lg-8 mx-auto text-center">
	  <div class="large text-muted">
</div>
	</div>
  </div>
</div>
</section>
<!-- End Team -->

<!-- About -->

<!-- End About -->

<!-- Contact -->
  <!-- <section class="page-section" id="contactZH">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 text-center" >
          <h2 class="section-heading text-uppercase">
</h2>
          <h3 class="section-subheading" style='font-style: normal;'><b>邮箱</b> : jian_zhang@tju.edu.cn</h3>
          <h3 class="section-subheading_next" style='font-style: normal;'><b>地址</b> : 天津市南开区卫津路92号</h3>
        </div>
      </div>
      <div class="row">
        <div class="col-lg-12">
          <form action="https://formspree.io/f/xbjpqwda" method="POST">

            <div class="row">
              <div class="col-md-6">
                <div class="form-group">
                  <input name="name" class="form-control" id="name" type="text" placeholder="Name*" required="required" data-validation-required-message="Please enter your name.">
                  <p class="help-block text-danger"></p>
                </div>
                <div class="form-group">
                  <input name="_replyto" class="form-control" id="email" type="email" placeholder="Email*" required="required" data-validation-required-message="Please enter your email address.">
                  <p class="help-block text-danger"></p>
                </div>
              </div>
              <div class="col-md-6">
                <div class="form-group">
                  <textarea name="message" class="form-control" id="message" placeholder="Message*" required="required" data-validation-required-message="Please enter a message."></textarea>
                  <p class="help-block text-danger"></p>
                </div>
              </div>
			  <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
              <div class="clearfix"></div>
              <div class="col-lg-12 text-center">
                <div id="success"></div>
                <button type="submit" class="btn btn-primary btn-xl text-uppercase" type="submit">发送邮件</button>
              </div>
			  <input type="text" name="_gotcha" style="display:none">
			  <input type="hidden" name="_next" value="#"/>
            </div>
          </form>
        </div>
      </div>
    </div>
  </section> -->
<!-- End Contact -->



	    <!-- Footer -->
  <footer class="bg-light footer" id="footer" style="background-color:white;">


    <div style='height: 300px;/* width:60%; *//* margin-left:20%; *//* z-index:99; *//* position:relative; */'>
<div class="container">
  <div class="row">
    <div class="col-8"  style='padding:15px;' >
        <h2 class="section-heading text-uppercase"  style='padding:15px' align='left'>CONTACT US</h2>
      <p align='left'><i class='fa fa-location-arrow' ></i> No.92 Weijin Road Nankai District, Tianjin, China</p>
      <p align='left'><i class='fa fa-envelope' ></i> jian_zhang@tju.edu.cn </p>
      <p align='left'><a style='color:#000;text-decoration:none'  href='https://scholar.google.com/citations?user=15wquMIAAAAJ&hl=zh-CN&oi=sra'><i class="fab fa-google" ></i> Google Scholar</a></p>
    </div>

    <div class="col-4" style='height:300px;line-height: 300px;'>
      <a href="http://www.tju.edu.cn/" target="_blank">
      <img class="mx-auto rounded-circle" src="assets/img/1.png" alt="" ></a>
      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
      <a href="http://mctu.tju.edu.cn/" target="_blank">
        <img class="mx-auto rounded-circle" src="assets/img/2.png" alt="" ></a>
    </div>
  </div>
</div>




<div class='bg-light'>
<ul style='color:#666;letter-spacing:.025em;line-height:1.5rem;font-size:1rem;font-weight:400;opacity:.7'>
  <span>&copy; Zhang Lab, 2022</span><br>
</ul></div>
    </div>

  </footer>

  <!-- End Footer -->


	  <!-- Bootstrap core JavaScript -->
	  <script src="assets/js/jquery.min.js"></script>
	  <script src="assets/js/bootstrap.bundle.min.js"></script>

	  <!-- Plugin JavaScript -->
	  <script src="assets/js/jquery.easing.min.js"></script>

	  <!-- Contact form JavaScript -->
	  <script src="assets/js/jqBootstrapValidation.js"></script>
	  <script src="assets/js/contact_me.js"></script>

	  <!-- Custom scripts for this template -->
	  <script src="assets/js/agency.min.js"></script>

  </body>
</html>